src/sentry/static/sentry/app/views/settings/settingsIndex.tsx

```diff
@@ -0,0 +1,388 @@
+import DocumentTitle from 'react-document-title';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled, {css} from 'react-emotion';
+
+import {t} from 'app/locale';
```

Comment:
Does your import js work with ts @EvanPurkhiser 	?

Reply:
Hmm, maybe. I accidentally deleted my config again the other day, so def not right now